app/controllers/api/v1/private/publishers_controller.rb

```diff
@@ -0,0 +1,78 @@
+class Api::V1::Private::PublishersController < Api::V1::Private::BaseController
+  class InvalidNote < StandardError; end
+  class InvalidAdmin < StandardError; end
+  def show
+    publisher = Publisher.find(params[:id])
+    current_status_update = publisher.last_status_update
+    data = {
+      id: publisher.id,
+      owner_identifier: publisher.owner_identifier,
+      email: publisher.email,
+      name: publisher.name,
+      created_at: publisher.created_at,
+      current_status: {
+        status: current_status_update.status,
+        created_at: current_status_update.created_at,
+      },
+    }
+    render(status: 200, json: data) and return
+
+  rescue ActiveRecord::RecordNotFound
+    error_response = {
+      errors: [{
+        status: "404",
+        title: "Not Found",
+        detail: "Publisher with id #{params[:publisher_id]} not found",
+      },],
+    }
+
+    render(status: 404, json: error_response) and return
+  end
+
+  def publisher_status_updates
+    publisher = Publisher.find(params[:publisher_id])
+    admin = Publisher.find_by_email(params[:admin])
```

Comment:
Let's name this `user` instead

Reply:
Sounds good, done in 0f23bb9

Reply:
No, sorry, I should have been more clear, I was thinking we should name the `admin` param `email` since it is the email of the administrator, that way it would be more clear 🤔

Although now i'm not too sure, what do you think?